src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  background-image: url('images/star.svg');
+  background-size: cover;
```

Comment:
don't use cover because as you can see star doesn't touch edges of it's container instead try to center it's position 
![image](https://github.com/user-attachments/assets/8842d882-770a-45a9-b04c-7cd353a49018)

Reply:
I have done new commit however it still doesn't look like on test file ;( it's close but not perfect :)